File: cfgs/java_cfgs_foo/SMALLEST_OF_THREE_INTEGERS_WITHOUT_COMPARISON_OPERATORS.java
Nodes
1;special;;;
2;statement;int c = 0;14;
3;conditional;((foo_1 != 0) && (foo_2 != 0)) && (foo_3 != 0);15;
4;statement;x--;16;
5;statement;y--;17;
6;statement;z--;18;
7;statement;c++;19;
8;statement;return foo_1;21;
9;exit;;;
Edges
0;2;
1;3;
2;8;4;
3;5;
4;6;
5;7;
6;3;
7;9;
8;

File: cfgs/pythoncfgs/SMALLEST_OF_THREE_INTEGERS_WITHOUT_COMPARISON_OPERATORS.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;foo_1 and foo_2 and foo_3;9;
3;statement;return foo_1;14;
4;statement;foo_1 = foo_2 - 1;10;
5;statement;foo_1 = foo_2 - 1;11;
6;statement;foo_1 = foo_2 - 1;12;
7;statement;foo_1 = foo_2 + 1;13;
8;exit;;
Edges
0;2;
1;3;4;
2;8;
3;5;
4;6;
5;7;
6;2;
7;